gammapy/scripts/fermi_3FGL_info.py

```diff
@@ -0,0 +1,32 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from __future__ import print_function, division
+import logging
+logging.basicConfig(level=logging.INFO, format='%(levelname)s - %(message)s')
+import os
+from astropy.units import Quantity
+from astropy.io import fits
+from ..utils.scripts import get_parser
+
+__all__ = ['fermi_3FGL_info']
+
+
+def print_info(source_name):
```

Comment:
Sorry if I'm asking obvious questions here.   Do you mean I should make a file gammapy/scripts/foo.py which creates a parser?

What should the Fermi3FGLObject.info method do?  Print values?  Return them?